pkg/database/migrations.go

```diff
@@ -2269,6 +2269,25 @@ func (db *Database) Migrations(ctx context.Context) []*gormigrate.Migration {
 					`DROP TABLE secrets`)
 			},
 		},
+		{
+			ID: "00104-AddCodeOverrides",
+			Migrate: func(tx *gorm.DB) error {
+				return multiExec(tx,
+					`ALTER TABLE realms
+						ADD COLUMN IF NOT EXISTS short_code_max_minutes SMALLINT DEFAULT 60,
+						ADD COLUMN IF NOT EXISTS enx_code_expiration_configurable BOOL DEFAULT false`,
+					`ALTER TABLE realms
+						ALTER COLUMN short_code_max_minutes SET NOT NULL,
```

Comment:
Need to SET DEFAULT in another stanza (no default will be set if the column already exists)

Reply:
we've had this exact discussion before :)
can be same TX, but different statement.

Reply:
Right, different statement is what I meant.

Reply:
This works as is for both a new and upgraded DB (tested both)